test/functional/wallet_multisig_descriptor_psbt.py

```diff
@@ -139,12 +143,12 @@ def run_test(self):
 
         self.log.info("Send another transaction from the multisig, this time with a daisy chained signing flow (one after another in series)!")
         psbt = participants["multisigs"][0].walletcreatefundedpsbt(inputs=[], outputs={to: value}, feeRate=0.00010)
-        for m in range(self.M):
+        for i, m in enumerate(random.sample(range(self.M), self.M)):
```

Comment:
`m` could be different in each test run because it uses `random.sample` as I found out while debugging this tst.. For thoroughness, would it be more robust to cover all the orderings instead? In this case it's going to be 2 only because `self.M = 2`.

Reply:
In this case `random.sample` shuffles the order of participants, but I don't see how it matters since all participants are equivalent anyway

Reply:
>  but I don't see how it matters since all participants are equivalent anyway

exactly, all participants are equivalent, and the intention of this `random.sample` is to prove that. maybe it's a little extra to even have this `random.sample`, but participants being equivalent is critical to the intention of this multisig so I thought the randomness was a nice to have